config-model-generator/pom.xml

```diff
@@ -24,12 +24,10 @@
         <dependency>
             <groupId>org.apache.kafka</groupId>
             <artifactId>kafka_2.12</artifactId>
-            <version>${kafka-metadata-version}</version>
```

Comment:
I think these were definitely intentionally there.

Reply:
Thanks for spotting this - I added a management section in this module with a comment to support the overrides, let me know what you think about that approach. Otherwise, I'm happy to restore this to the original and ignore Eclipse's warnings if it's too much :)